[ihc] Tests failing on Windows

Run on Windows 10:
```
[INFO] --- maven-surefire-plugin:2.22.1:test (default-test) @ org.openhab.binding.ihc ---
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.binding.ihc.internal.ButtonPressDurationDetectorTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.047 s - in org.openhab.binding.ihc.internal.ButtonPressDurationDetectorTest
[INFO] Running org.openhab.binding.ihc.internal.converters.DateTimeTypeWSDateValueConverterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.031 s - in org.openhab.binding.ihc.internal.converters.DateTimeTypeWSDateValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.DateTimeTypeWSTimeValueConverterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.DateTimeTypeWSTimeValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.DecimalTypeWSBooleanValueConverterTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.DecimalTypeWSBooleanValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.DecimalTypeWSEnumValueConverterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.DecimalTypeWSEnumValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.DecimalTypeWSFloatingPointValueConverterTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.DecimalTypeWSFloatingPointValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.DecimalTypeWSIntegerValueConverterTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.DecimalTypeWSIntegerValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.DecimalTypeWSTimerValueConverterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.DecimalTypeWSTimerValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.DecimalTypeWSWeekdayValueConverterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.DecimalTypeWSWeekdayValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.OnOffTypeWSBooleanValueConverterTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.OnOffTypeWSBooleanValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.OnOffTypeWSIntegerValueConverterTest
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.OnOffTypeWSIntegerValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.OpenClosedTypeWSBooleanValueConverterTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.OpenClosedTypeWSBooleanValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.OpenClosedTypeWSIntegerValueConverterTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.OpenClosedTypeWSIntegerValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.PercentTypeWSIntegerValueConverterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.PercentTypeWSIntegerValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.StringTypeWSEnumValueConverterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.StringTypeWSEnumValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.UpDownTypeWSBooleanValueConverterTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.UpDownTypeWSBooleanValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.converters.UpDownTypeWSIntegerValueConverterTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.converters.UpDownTypeWSIntegerValueConverterTest
[INFO] Running org.openhab.binding.ihc.internal.SignalLevelConverterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.ihc.internal.SignalLevelConverterTest
[INFO] Running org.openhab.binding.ihc.internal.ws.IhcClientTest
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.297 s <<< FAILURE! - in org.openhab.binding.ihc.internal.ws.IhcClientTest
[ERROR] loadProjectFileFromControllerTest(org.openhab.binding.ihc.internal.ws.IhcClientTest)  Time elapsed: 0.297 s  <<< FAILURE!
org.junit.ComparisonFailure:
expected:<...onsectetur sagittis.[

Sed hendrerit, est eu hendrerit condimentum, mi justo vehicula ipsum, at dignissim tortor mi quis velit. Aliquam in turpis quis mi dapibus efficitur sed vel metus. Donec ac nisl magna. Cras eget nulla in nunc mollis semper quis et ex. Nulla accumsan dui non mi interdum interdum a nec justo. Phasellus at dolor sit amet purus porta tristique eget in diam. Suspendisse laoreet vulputate nulla quis dictum. Sed laoreet elit sed ipsum ornare ornare. Vivamus in blandit enim, et luctus lacus. Aenean id laoreet diam. Pellentesque ipsum diam, fermentum nec ante ac, mattis tristique mi. Curabitur ultricies risus cursus iaculis ultrices. Nullam sed pharetra arcu. Sed sed dui congue, porttitor nisl vitae, viverra risus.
]
Proin at lorem non ...> but was:<...onsectetur sagittis.[

Sed hendrerit, est eu hendrerit condimentum, mi justo vehicula ipsum, at dignissim tortor mi quis velit. Aliquam in turpis quis mi dapibus efficitur sed vel metus. Donec ac nisl magna. Cras eget nulla in nunc mollis semper quis et ex. Nulla accumsan dui non mi interdum interdum a nec justo. Phasellus at dolor sit amet purus porta tristique eget in diam. Suspendisse laoreet vulputate nulla quis dictum. Sed laoreet elit sed ipsum ornare ornare. Vivamus in blandit enim, et luctus lacus. Aenean id laoreet diam. Pellentesque ipsum diam, fermentum nec ante ac, mattis tristique mi. Curabitur ultricies risus cursus iaculis ultrices. Nullam sed pharetra arcu. Sed sed dui congue, porttitor nisl vitae, viverra risus.
]
Proin at lorem non ...>
        at org.openhab.binding.ihc.internal.ws.IhcClientTest.loadProjectFileFromControllerTest(IhcClientTest.java:64)

[INFO] Running org.openhab.binding.ihc.internal.ws.services.IhcAirlinkManagementServiceTest
log4j:WARN No appenders could be found for logger (org.apache.http.client.protocol.RequestAddCookies).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.047 s <<< FAILURE! - in org.openhab.binding.ihc.internal.ws.services.IhcAirlinkManagementServiceTest
[ERROR] test(org.openhab.binding.ihc.internal.ws.services.IhcAirlinkManagementServiceTest)  Time elapsed: 1.047 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcAirlinkManagementServiceTest.test(IhcAirlinkManagementServiceTest.java:60)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcAirlinkManagementServiceTest.test(IhcAirlinkManagementServiceTest.java:60)

[INFO] Running org.openhab.binding.ihc.internal.ws.services.IhcAuthenticationServiceTest
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.703 s <<< FAILURE! - in org.openhab.binding.ihc.internal.ws.services.IhcAuthenticationServiceTest
[ERROR] testFailedLogin(org.openhab.binding.ihc.internal.ws.services.IhcAuthenticationServiceTest)  Time elapsed: 0.344 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcAuthenticationServiceTest.testFailedLogin(IhcAuthenticationServiceTest.java:68)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcAuthenticationServiceTest.testFailedLogin(IhcAuthenticationServiceTest.java:68)

[ERROR] testSuccesfulLogin(org.openhab.binding.ihc.internal.ws.services.IhcAuthenticationServiceTest)  Time elapsed: 0.359 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcAuthenticationServiceTest.testSuccesfulLogin(IhcAuthenticationServiceTest.java:59)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcAuthenticationServiceTest.testSuccesfulLogin(IhcAuthenticationServiceTest.java:59)

[INFO] Running org.openhab.binding.ihc.internal.ws.services.IhcConfigurationServiceTest
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.734 s <<< FAILURE! - in org.openhab.binding.ihc.internal.ws.services.IhcConfigurationServiceTest
[ERROR] testv2(org.openhab.binding.ihc.internal.ws.services.IhcConfigurationServiceTest)  Time elapsed: 0.36 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcConfigurationServiceTest.testv2(IhcConfigurationServiceTest.java:59)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcConfigurationServiceTest.testv2(IhcConfigurationServiceTest.java:59)

[ERROR] testv3(org.openhab.binding.ihc.internal.ws.services.IhcConfigurationServiceTest)  Time elapsed: 0.374 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcConfigurationServiceTest.testv3(IhcConfigurationServiceTest.java:82)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcConfigurationServiceTest.testv3(IhcConfigurationServiceTest.java:82)

[INFO] Running org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest
[ERROR] Tests run: 6, Failures: 0, Errors: 6, Skipped: 0, Time elapsed: 1.937 s <<< FAILURE! - in org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest
[ERROR] projectNumberOfSegmentsTest(org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest)  Time elapsed: 0.235 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.projectNumberOfSegmentsTest(IhcControllerServiceTest.java:78)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.projectNumberOfSegmentsTest(IhcControllerServiceTest.java:78)

[ERROR] projectInfoTest(org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest)  Time elapsed: 0.343 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.projectInfoTest(IhcControllerServiceTest.java:64)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.projectInfoTest(IhcControllerServiceTest.java:64)

[ERROR] projectSegmentationSizeTest(org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest)  Time elapsed: 0.359 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.projectSegmentationSizeTest(IhcControllerServiceTest.java:93)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.projectSegmentationSizeTest(IhcControllerServiceTest.java:93)

[ERROR] controllerStateTest(org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest)  Time elapsed: 0.36 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.controllerStateTest(IhcControllerServiceTest.java:106)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.controllerStateTest(IhcControllerServiceTest.java:106)

[ERROR] projectSegmentQueryTest(org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest)  Time elapsed: 0.359 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.projectSegmentQueryTest(IhcControllerServiceTest.java:139)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.projectSegmentQueryTest(IhcControllerServiceTest.java:139)

[ERROR] waitForControllerStateChangeQueryTest(org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest)  Time elapsed: 0.281 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Encoding error, unsupported data
        at org.openhab.binding.ihc.internal.ws.services.IhcControllerServiceTest.waitForControllerStateChangeQueryTest(IhcControllerServiceTest.java:124)

[INFO] Running org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[Fatal Error] :6:3: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
[ERROR] Tests run: 18, Failures: 0, Errors: 18, Skipped: 0, Time elapsed: 6.202 s <<< FAILURE! - in org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest
[ERROR] testWSWeekdayValueQuery(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.297 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSWeekdayValueQuery(IhcResourceInteractionServiceTest.java:165)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSWeekdayValueQuery(IhcResourceInteractionServiceTest.java:165)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSWeekdayValueQuery(IhcResourceInteractionServiceTest.java:165)

[ERROR] testWSTimeValueQuery(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.734 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimeValueQuery(IhcResourceInteractionServiceTest.java:181)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimeValueQuery(IhcResourceInteractionServiceTest.java:181)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimeValueQuery(IhcResourceInteractionServiceTest.java:181)

[ERROR] testWSBooleanValueQuery(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.297 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueQuery(IhcResourceInteractionServiceTest.java:124)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueQuery(IhcResourceInteractionServiceTest.java:124)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueQuery(IhcResourceInteractionServiceTest.java:124)

[ERROR] testWSEnumValueUpdate(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.281 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSEnumValueUpdate(IhcResourceInteractionServiceTest.java:209)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSEnumValueUpdate(IhcResourceInteractionServiceTest.java:209)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSEnumValueUpdate(IhcResourceInteractionServiceTest.java:209)

[ERROR] testWSDateValueUpdate(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.281 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSDateValueUpdate(IhcResourceInteractionServiceTest.java:234)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSDateValueUpdate(IhcResourceInteractionServiceTest.java:234)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSDateValueUpdate(IhcResourceInteractionServiceTest.java:234)

[ERROR] testWSTimerValueUpdate(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.281 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimerValueUpdate(IhcResourceInteractionServiceTest.java:221)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimerValueUpdate(IhcResourceInteractionServiceTest.java:221)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimerValueUpdate(IhcResourceInteractionServiceTest.java:221)

[ERROR] testWSIntegerValueUpdate(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.313 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSIntegerValueUpdate(IhcResourceInteractionServiceTest.java:215)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSIntegerValueUpdate(IhcResourceInteractionServiceTest.java:215)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSIntegerValueUpdate(IhcResourceInteractionServiceTest.java:215)

[ERROR] testWSTimeValueUpdate(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.281 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimeValueUpdate(IhcResourceInteractionServiceTest.java:240)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimeValueUpdate(IhcResourceInteractionServiceTest.java:240)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimeValueUpdate(IhcResourceInteractionServiceTest.java:240)

[ERROR] testWSFloatingPointValueUpdate(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.281 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSFloatingPointValueUpdate(IhcResourceInteractionServiceTest.java:203)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSFloatingPointValueUpdate(IhcResourceInteractionServiceTest.java:203)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSFloatingPointValueUpdate(IhcResourceInteractionServiceTest.java:203)

[ERROR] testWSIntegerValueQuery(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.297 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSIntegerValueQuery(IhcResourceInteractionServiceTest.java:149)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSIntegerValueQuery(IhcResourceInteractionServiceTest.java:149)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSIntegerValueQuery(IhcResourceInteractionServiceTest.java:149)

[ERROR] testWSEnumValueQuery(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.281 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSEnumValueQuery(IhcResourceInteractionServiceTest.java:140)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSEnumValueQuery(IhcResourceInteractionServiceTest.java:140)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSEnumValueQuery(IhcResourceInteractionServiceTest.java:140)

[ERROR] testWSFloatingPointValueQuery(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.281 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSFloatingPointValueQuery(IhcResourceInteractionServiceTest.java:131)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSFloatingPointValueQuery(IhcResourceInteractionServiceTest.java:131)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSFloatingPointValueQuery(IhcResourceInteractionServiceTest.java:131)

[ERROR] testWSTimerValueQuery(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.297 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimerValueQuery(IhcResourceInteractionServiceTest.java:158)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimerValueQuery(IhcResourceInteractionServiceTest.java:158)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSTimerValueQuery(IhcResourceInteractionServiceTest.java:158)

[ERROR] testResourceValueNotifications(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.75 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testResourceValueNotifications(IhcResourceInteractionServiceTest.java:246)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testResourceValueNotifications(IhcResourceInteractionServiceTest.java:246)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testResourceValueNotifications(IhcResourceInteractionServiceTest.java:246)

[ERROR] testWSBooleanValueUpdate(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.312 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueUpdate(IhcResourceInteractionServiceTest.java:190)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueUpdate(IhcResourceInteractionServiceTest.java:190)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueUpdate(IhcResourceInteractionServiceTest.java:190)

[ERROR] testWSDateValueQuery(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.329 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: Error occured during XML data parsing
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSDateValueQuery(IhcResourceInteractionServiceTest.java:172)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSDateValueQuery(IhcResourceInteractionServiceTest.java:172)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSDateValueQuery(IhcResourceInteractionServiceTest.java:172)

[ERROR] testWSBooleanValueUpdateFailure(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.312 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueUpdateFailure(IhcResourceInteractionServiceTest.java:196)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueUpdateFailure(IhcResourceInteractionServiceTest.java:196)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSBooleanValueUpdateFailure(IhcResourceInteractionServiceTest.java:196)

[ERROR] testWSWeekdayValueUpdate(org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest)  Time elapsed: 0.297 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSWeekdayValueUpdate(IhcResourceInteractionServiceTest.java:227)
Caused by: javax.xml.xpath.XPathExpressionException: org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 3; Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSWeekdayValueUpdate(IhcResourceInteractionServiceTest.java:227)
Caused by: org.xml.sax.SAXParseException: Le type d'▒l▒ment "hr" doit se terminer par la balise de fin correspondante "</hr>".
        at org.openhab.binding.ihc.internal.ws.services.IhcResourceInteractionServiceTest.testWSWeekdayValueUpdate(IhcResourceInteractionServiceTest.java:227)

[INFO] Running org.openhab.binding.ihc.internal.ws.services.IhcTimeServiceTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.344 s <<< FAILURE! - in org.openhab.binding.ihc.internal.ws.services.IhcTimeServiceTest
[ERROR] test(org.openhab.binding.ihc.internal.ws.services.IhcTimeServiceTest)  Time elapsed: 0.344 s  <<< ERROR!
org.openhab.binding.ihc.internal.ws.exeptions.IhcExecption: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcTimeServiceTest.test(IhcTimeServiceTest.java:60)
Caused by: java.net.SocketTimeoutException: Read timed out
        at org.openhab.binding.ihc.internal.ws.services.IhcTimeServiceTest.test(IhcTimeServiceTest.java:60)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   IhcClientTest.loadProjectFileFromControllerTest:64 expected:<...onsectetur sagittis.[

Sed hendrerit, est eu hendrerit condimentum, mi justo vehicula ipsum, at dignissim tortor mi quis velit. Aliquam in turpis quis mi dapibus efficitur sed vel metus. Donec ac nisl magna. Cras eget nulla in nunc mollis semper quis et ex. Nulla accumsan dui non mi interdum interdum a nec justo. Phasellus at dolor sit amet purus porta tristique eget in diam. Suspendisse laoreet vulputate nulla quis dictum. Sed laoreet elit sed ipsum ornare ornare. Vivamus in blandit enim, et luctus lacus. Aenean id laoreet diam. Pellentesque ipsum diam, fermentum nec ante ac, mattis tristique mi. Curabitur ultricies risus cursus iaculis ultrices. Nullam sed pharetra arcu. Sed sed dui congue, porttitor nisl vitae, viverra risus.
]
Proin at lorem non ...> but was:<...onsectetur sagittis.[

Sed hendrerit, est eu hendrerit condimentum, mi justo vehicula ipsum, at dignissim tortor mi quis velit. Aliquam in turpis quis mi dapibus efficitur sed vel metus. Donec ac nisl magna. Cras eget nulla in nunc mollis semper quis et ex. Nulla accumsan dui non mi interdum interdum a nec justo. Phasellus at dolor sit amet purus porta tristique eget in diam. Suspendisse laoreet vulputate nulla quis dictum. Sed laoreet elit sed ipsum ornare ornare. Vivamus in blandit enim, et luctus lacus. Aenean id laoreet diam. Pellentesque ipsum diam, fermentum nec ante ac, mattis tristique mi. Curabitur ultricies risus cursus iaculis ultrices. Nullam sed pharetra arcu. Sed sed dui congue, porttitor nisl vitae, viverra risus.
]
Proin at lorem non ...>
[ERROR] Errors:
[ERROR]   IhcAirlinkManagementServiceTest.test:60 ▒ IhcExecption java.net.SocketTimeoutE...
[ERROR]   IhcAuthenticationServiceTest.testFailedLogin:68 ▒ IhcExecption java.net.Socket...
[ERROR]   IhcAuthenticationServiceTest.testSuccesfulLogin:59 ▒ IhcExecption java.net.Soc...
[ERROR]   IhcConfigurationServiceTest.testv2:59 ▒ IhcExecption java.net.SocketTimeoutExc...
[ERROR]   IhcConfigurationServiceTest.testv3:82 ▒ IhcExecption java.net.SocketTimeoutExc...
[ERROR]   IhcControllerServiceTest.controllerStateTest:106 ▒ IhcExecption java.net.Socke...
[ERROR]   IhcControllerServiceTest.projectInfoTest:64 ▒ IhcExecption java.net.SocketTime...
[ERROR]   IhcControllerServiceTest.projectNumberOfSegmentsTest:78 ▒ IhcExecption java.ne...
[ERROR]   IhcControllerServiceTest.projectSegmentQueryTest:139 ▒ IhcExecption java.net.S...
[ERROR]   IhcControllerServiceTest.projectSegmentationSizeTest:93 ▒ IhcExecption java.ne...
[ERROR]   IhcControllerServiceTest.waitForControllerStateChangeQueryTest:124 ▒ IhcExecption
[ERROR]   IhcResourceInteractionServiceTest.testResourceValueNotifications:246 ▒ IhcExecption
[ERROR]   IhcResourceInteractionServiceTest.testWSBooleanValueQuery:124 ▒ IhcExecption E...
[ERROR]   IhcResourceInteractionServiceTest.testWSBooleanValueUpdate:190 ▒ IhcExecption ...
[ERROR]   IhcResourceInteractionServiceTest.testWSBooleanValueUpdateFailure:196 ▒ IhcExecption
[ERROR]   IhcResourceInteractionServiceTest.testWSDateValueQuery:172 ▒ IhcExecption Erro...
[ERROR]   IhcResourceInteractionServiceTest.testWSDateValueUpdate:234 ▒ IhcExecption jav...
[ERROR]   IhcResourceInteractionServiceTest.testWSEnumValueQuery:140 ▒ IhcExecption Erro...
[ERROR]   IhcResourceInteractionServiceTest.testWSEnumValueUpdate:209 ▒ IhcExecption jav...
[ERROR]   IhcResourceInteractionServiceTest.testWSFloatingPointValueQuery:131 ▒ IhcExecption
[ERROR]   IhcResourceInteractionServiceTest.testWSFloatingPointValueUpdate:203 ▒ IhcExecption
[ERROR]   IhcResourceInteractionServiceTest.testWSIntegerValueQuery:149 ▒ IhcExecption E...
[ERROR]   IhcResourceInteractionServiceTest.testWSIntegerValueUpdate:215 ▒ IhcExecption ...
[ERROR]   IhcResourceInteractionServiceTest.testWSTimeValueQuery:181 ▒ IhcExecption Erro...
[ERROR]   IhcResourceInteractionServiceTest.testWSTimeValueUpdate:240 ▒ IhcExecption jav...
[ERROR]   IhcResourceInteractionServiceTest.testWSTimerValueQuery:158 ▒ IhcExecption Err...
[ERROR]   IhcResourceInteractionServiceTest.testWSTimerValueUpdate:221 ▒ IhcExecption ja...
[ERROR]   IhcResourceInteractionServiceTest.testWSWeekdayValueQuery:165 ▒ IhcExecption E...
[ERROR]   IhcResourceInteractionServiceTest.testWSWeekdayValueUpdate:227 ▒ IhcExecption ...
[ERROR]   IhcTimeServiceTest.test:60 ▒ IhcExecption java.net.SocketTimeoutException: Rea...
[INFO]
[ERROR] Tests run: 77, Failures: 1, Errors: 30, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 20.880 s
[INFO] Finished at: 2019-08-10T08:17:42+02:00
[INFO] Final Memory: 29M/718M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.1:test (default-test) on project org.openhab.binding.ihc: There are test failures.
```

--- a/bundles/org.openhab.binding.ihc/src/test/java/org/openhab/binding/ihc/internal/ws/ResourceFileUtils.java
+++ b/bundles/org.openhab.binding.ihc/src/test/java/org/openhab/binding/ihc/internal/ws/ResourceFileUtils.java
@@ -14,9 +14,12 @@ package org.openhab.binding.ihc.internal.ws;
 
 import static org.junit.Assert.fail;
 
+import java.io.BufferedReader;
 import java.io.IOException;
-
-import org.apache.commons.io.IOUtils;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.stream.Collectors;
 
 /**
  * Util class to load file content from resource files.
@@ -25,12 +28,13 @@ import org.apache.commons.io.IOUtils;
  */
 public class ResourceFileUtils {
     static public String getFileContent(String resourceFile) {
-        String result = "";
-        try {
-            result = IOUtils.toString(ResourceFileUtils.class.getClassLoader().getResourceAsStream(resourceFile));
+        try (InputStream inputStream = ResourceFileUtils.class.getClassLoader().getResourceAsStream(resourceFile);
+                Reader inputStreamReader = new InputStreamReader(inputStream);
+                BufferedReader bufferedReader = new BufferedReader(inputStreamReader)) {
+            return bufferedReader.lines().collect(Collectors.joining("\n"));
         } catch (IOException e) {
             fail("IOException reading xml file '" + resourceFile + "': " + e);
         }
-        return result;
+        return "";
     }
 }

